presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -1100,6 +1102,16 @@ public SystemSessionProperties(
                         AGGREGATION_IF_TO_FILTER_REWRITE_ENABLED,
                         "Enable rewriting the IF expression inside an aggregation function to a filter clause outside the aggregation",
                         featuresConfig.isAggregationIfToFilterRewriteEnabled(),
+                        false),
+                booleanProperty(
+                        HEAP_DUMP_ON_EXCEEDED_MEMORY_LIMIT_ENABLED,
+                        "Trigger heap dump to `EXCEEDED_MEMORY_LIMIT_HEAP_DUMP_FILE_PATH` on exceeded memory limit exceptions",
+                        false,
+                        false),
```

Comment:
we probably don't wanna expose this session property to users..... let's make it hidden; same for the other one